test/e2e/framework/framework.go

```diff
@@ -92,8 +95,8 @@ func NewFramework() *Framework {
 }
 
 func init() {
-	homeDir := GetHomeDir()
-	TestDirPath = filepath.Join(homeDir, TestDir)
+	OriginalHomeDir = GetHomeDir()
+	TestDirPath = filepath.Join(OriginalHomeDir)
```

Comment:
Why are we no longer adding `TestDir` in this `Join`?

Reply:
Thanks for catching this. I forgot to revert this change back after overwriting the `HOME` variable on the specific test.